go.mod

```diff
@@ -20,14 +20,14 @@ require (
 	go.uber.org/zap v1.20.0
 	golang.org/x/net v0.0.0-20220127200216-cd36cc0744dd
 	golang.org/x/sys v0.0.0-20220209214540-3681064d5158 // indirect
-	google.golang.org/genproto v0.0.0-20220211171837-173942840c17 // indirect
+	google.golang.org/genproto v0.0.0-20220215190005-e57b466719ef // indirect
```

Comment:
This should be `github.com/instill-ai/protogen-go v0.0.0-20220216035831-2d9577e0f42b`

Reply:
The version is still not the up-to-date one

Reply:
oh, my mistake, that's not our package